data/blog/libtailscale.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: "libtailscale: how it's made"
+date: '2023-03-02'
+tags:
+  - 'tsnet'
+  - 'libtailscale'
+  - 'oss'
+authors: ['crawshaw']
+summary: 'libtailscale: a Tailscale C library'
+---
+
+We just released an experimental libtailscale, the Tailscale C library.
+It is built on the Go [tsnet](https://pkg.go.dev/tailscale.com/tsnet)
+package which uses a userland network stack to implement TCP/IP
+directly inside your process.
+
+```
```

Comment:
For syntax highlighting: \```c

Reply:
Offer a description of the sample?

```// an ephemeral tailnet echo server```

I'm also thinking of netcat and that cats have tails but I puns are failing me